src/js/smooch.jsx

```diff
@@ -16,12 +16,12 @@ import { reset } from 'actions/common-actions';
 import { openWidget, closeWidget } from 'services/app-service';
 import { login } from 'services/auth-service';
 import { getAccount } from 'services/stripe-service';
-import { EDITABLE_PROPERTIES, trackEvent, update as updateUser, immediateUpdate as immediateUpdateUser } from 'services/user-service';
+import { EDITABLE_PROPERTIES, trackEvent, update as updateUser, updateNowViewing, immediateUpdate as immediateUpdateUser } from 'services/user-service';
 import { getConversation, sendMessage, connectFaye, disconnectFaye, handleConversationUpdated } from 'services/conversation-service';
 
 import { observable } from 'utils/events';
 import { storage } from 'utils/storage';
-import { waitForPage } from 'utils/dom';
+import { waitForPage, monitorUrlChanges, stopMonitoring as stopMonitoringUrlChanges } from 'utils/dom';
```

Comment:
Why not call `stopMonitoringUrlChanges` in the export instead of changing the name when you import it?